insights/parsers/gluster_vol.py

```diff
@@ -84,3 +84,72 @@ def parse_content(self, content):
             # object.  Rules depending solely on this parser will not be invoked, so they don't have to
             # explicitly check for invalid data.
             raise ParseException("Unable to parse gluster volume options: {0}".format(content))
+
+
+@parser(Specs.gluster_v_status)
+class GlusterVolStatus(LegacyItemAccess, CommandParser):
+    """
+    This parser processes the output of the command `gluster vol status` and provides
+    the information as a dictionary.
+
+    The LegacyItemAccess class provides some helper functions for dealing with a
+    class having a `data` attribute.
+
+    Sample input::
+
+        Status of volume: test_vol
+        Gluster process                             TCP Port  RDMA Port  Online  Pid
+        ------------------------------------------------------------------------------
+        Brick 172.17.18.42:/home/brick              49152     0          Y       26685
+        Brick 172.17.18.43:/home/brick              49152     0          Y       27094
+        Brick 172.17.18.44:/home/brick              49152     0          Y       27060
+        Self-heal Daemon on localhost               N/A       N/A        Y       7805
+        Self-heal Daemon on 172.17.18.44            N/A       N/A        Y       33400
+        Self-heal Daemon on 172.17.18.43            N/A       N/A        Y       33680
+
+        Task Status of Volume test_vol
+        ------------------------------------------------------------------------------
+        There are no active volume tasks
+
+
+    Examples:
+
+        >>> parser_result.data["test_vol"][0]
```

Comment:
As described in line-95, better access its content without the `.data` in the example.

And it would be helpful if you can add a test for the example case, like https://github.com/RedHatInsights/insights-core/blob/98aa6eb3f00b181f03b3075d1fba27554baecf89/insights/parsers/tests/test_ipcs.py#L108